docs/user/quickstart.rst

```diff
@@ -43,9 +43,9 @@ command line:
 Once Toolchest is configured, you can do the same with the corresponding
 Toolchest function:
 
->>> import toolchest_client as tc
->>> tc.your_tool(
-...     YOUR_CUSTOM_TOOL_ARGS,
+>>> import toolchest_client as toolchest
+>>> toolchest.your_tool(
+...     tool_args="YOUR_CUSTOM_TOOL_ARGS",
```

Comment:
@bcai2 I think those edits are in this PR already!

Reply:
Oops, gotcha